include/ace/managers/game.h

```diff
@@ -26,40 +24,60 @@ typedef struct _tGameState {
 	tGameCb cbCreate;
 	tGameCb cbLoop;
 	tGameCb cbDestroy;
-	struct _tGameState *pPrev;
+	tGameCb cbSuspend;
+	tGameCb cbResume;
+	struct _tGameState *pStatePrev;
 } tGameState;
 
 typedef struct _tGameManager {
-	UBYTE ubStateCount;
-	UBYTE isRunning;
-	tGameState *pStateFirst;
+	tGameState *pState;
```

Comment:
Same here. I think `pFirst` would be better but it's up to you.

Reply:
I'll stick with `pState`.

Reply:
wouldn't `pCurrent` be better?

Reply:
True, It will be more aligned with `pPrev`.